app/models/manageiq/providers/azure/cloud_manager/event_catcher/runner.rb

```diff
@@ -49,6 +45,8 @@ def reset_event_monitor_handle
 
   def filtered?(event)
     event_type = parse_event_type(event)
-    filtered_events.include?(event_type)
+    filtered_events.include?(event_type).tap do |result|
+      _log.info "#{log_prefix} Skipping filtered Azure event #{event_type} for #{event["resourceId"]}" if result
+    end
```

Comment:
I find this `.tap` style for this particular use case a bit confusing and hard to wrap my head around.
For checking a boolean value I'd rather go for a explicit syntax:

```ruby
if filtered_events.include?(event_type)
  _log.info "#{log_prefix} Skipping filtered Azure event #{event_type} for #{event["resourceId"]}"
  true
else
  false
end
```

But it might be just me :)